lib-install.sh

```diff
@@ -81,6 +81,8 @@ fi
     echo "user_pref(\"browser.tabs.drawInTitlebar\", true);" >>                         "${d}/prefs.js"
     echo "user_pref(\"browser.uidensity\", 0);" >>                                      "${d}/prefs.js"
     echo "user_pref(\"layers.acceleration.force-enabled\", true);" >>                   "${d}/prefs.js"
+    echo "user_pref(\"security.insecure_connection_text.enabled\", false);" >>          "${d}/prefs.js"
+    echo "user_pref(\"security.insecure_connection_text.pbmode.enabled\", false);" >>   "${d}/prefs.js"
```

Comment:
From what I understand here, this would hide the text here?
<img width="452" alt="image" src="https://github.com/user-attachments/assets/01bdf337-d1d0-476f-b923-96fd22dfe04a" />

Comparing with Safari there still is some text showing the connection
<img width="434" alt="image" src="https://github.com/user-attachments/assets/70bb8808-6fe3-4910-89d2-6523cbd88ce1" />

Would there be a reason to hide this?

Reply:
I find it unacceptable, btw.
There must be a better compromise rather than removing a security feature.
Ensuring the text areas don't overlap would be ideal.